logic/src/commonMain/kotlin/com/wire/kalium/logic/data/id/QualifiedId.kt

```diff
@@ -1,10 +1,23 @@
 package com.wire.kalium.logic.data.id
 
+private const val VALUE_DOMAIN_SEPARATOR = "@"
+
 data class QualifiedID(
     val value: String,
     val domain: String
 )
 
 typealias ConversationId = QualifiedID
 
-fun QualifiedID.asString() = "$value@$domain"
+fun QualifiedID.asString() = "$value$VALUE_DOMAIN_SEPARATOR$domain"
```

Comment:
We need to check if the domain is empty or not as well.
If we are not in federated environment, the domain will be empty
```
if(domain.isEmpty) return "$value"
else "$value$VALUE_DOMAIN_SEPARATOR$domain"
```